cdk/bin/security-vpc.ts

```diff
@@ -0,0 +1,14 @@
+import 'source-map-support/register';
+import { App } from '@aws-cdk/core';
+import { SecurityVpc } from '../lib/security-vpc';
+
+// Note, this stack is synthed locally rather than in CI as it is
+// 'environment-aware'
+// (https://docs.aws.amazon.com/cdk/latest/guide/environments.html) and so
+// requires AWS crendentials. Ideally, longer-term we'd support this in Github
+// Actions somehow.
+const app = new App();
+new SecurityVpc(app, 'security-vpc', {
+  stack: 'security',
+  env: { region: 'eu-west-1', account: process.env.CDK_DEFAULT_ACCOUNT }, // Crucial to ensure VPC uses all AZs.
```

Comment:
Any idea why this needs the account too? I would have hoped that the environment would have been enough 😞

Reply:
@jacobwinch unfortunately not - I was surprised by this too. To be 'environment-aware' requires both region and account number. (https://docs.aws.amazon.com/cdk/latest/guide/environments.html - I did try testing with just region too in case it worked but alas not.)

Reply:
That's a shame, thanks for explaining this though 👍